src/tyr/route_serializer_osrm.cc

```diff
@@ -334,8 +356,21 @@ json::ArrayPtr intersections(const valhalla::DirectionsLeg::Maneuver& maneuver,
     loc->emplace_back(json::fp_t{ll.lat(), 6});
     intersection->emplace("location", loc);
     intersection->emplace("geometry_index", static_cast<uint64_t>(shape_index));
-    if (node->has_transition_time())
-      intersection->emplace("duration", json::fp_t{node->transition_time(), 3});
+    if (node->cost().transition_cost().seconds() > 0)
+      intersection->emplace("turn_duration", json::fp_t{node->cost().transition_cost().seconds(), 3});
+    if (node->cost().transition_cost().cost() > 0)
+      intersection->emplace("turn_weight", json::fp_t{node->cost().transition_cost().cost(), 3});
```

Comment:
ive renamed the recently added "intersection.duration" which was a `transition_cost.secs` to "intersection.turn_duration".  so that i could also add the edges "duration" as the rest of the linear duration are named in the structure. im not particularly worried about this since its not documented (we have no osrm docs) nor is it in a versioned release of the library yet. let me know if we think this is a problem. i do really like having this information here though as it could be very useful for visual debugging of route paths since it includes edge and node level cost information in the response!